config/crds/azureprovider_v1alpha1_azureclusterproviderstatus.yaml

```diff
@@ -15,148 +15,179 @@ spec:
     openAPIV3Schema:
       properties:
         apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
           type: string
         bastion:
           properties:
-            ebsOptimized:
-              type: boolean
-            enaSupport:
-              type: boolean
-            iamProfile:
-              type: string
             id:
               type: string
-            imageId:
-              type: string
-            instanceState:
-              type: string
-            keyName:
+            identity:
               type: string
-            privateIp:
-              type: string
-            publicIp:
+            image:
+              description: Storage profile
+              properties:
+                offer:
+                  type: string
+                publisher:
+                  type: string
+                sku:
+                  type: string
+                version:
+                  type: string
+              required:
+              - publisher
+              - offer
+              - sku
+              - version
+              type: object
+            name:
               type: string
-            securityGroupIds:
-              items:
-                type: string
-              type: array
-            subnetId:
+            osDisk:
+              properties:
+                diskSizeGB:
+                  format: int32
+                  type: integer
+                managedDisk:
+                  properties:
+                    storageAccountType:
+                      type: string
+                  required:
+                  - storageAccountType
+                  type: object
+                osType:
+                  type: string
+              required:
+              - osType
+              - managedDisk
+              - diskSizeGB
+              type: object
+            startupScript:
               type: string
             tags:
               type: object
-            type:
+            vmSize:
+              description: Hardware profile
               type: string
-            userData:
+            vmState:
+              description: State - The provisioning state, which only appears in the
+                response.
               type: string
-          required:
-          - id
           type: object
         kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
```

Comment:
```suggestion
            submits requests to. Cannot be updated. (It is written in "camelCase") More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
```

Reply:
This is actually generated from `k8s.io/apimachinery/pkg/apis/meta/v1`: `TypeMeta.Kind`